src/browser/provider/built-in/path.js

```diff
@@ -56,5 +56,9 @@ export default {
 
     async isLocalBrowser () {
         return true;
-    }
+    },
+
+    isHeadlessBrowser () {
```

Comment:
You don't have to add this method to all providers, just add it to the https://github.com/DevExpress/testcafe/blob/master/src/browser/provider/plugin-host.js